source/slang/ir.cpp

```diff
@@ -423,6 +424,12 @@ namespace Slang
         return count;
     }
 
+    bool IRBlock::PredecessorList::isEmpty()
+    {
+        return !(begin() != end());
```

Comment:
so == then?

Reply:
I could add that missing operator, but the iterator type only defined `operator!=` right now, because that is the bare minimum required for an iterator.